java/SqsBatchProcessing/Functions/src/test/java/org/demo/sqs/SqsPollerTest.java

```diff
@@ -1,10 +1,13 @@
 package org.demo.sqs;
 
+import org.junit.Ignore;
 import org.junit.Test;
 
 public class SqsPollerTest {
 
+  // This test requires AWS access and cannot be run standalone like a unit test
   @Test
+  @Ignore
```

Comment:
I think that we can consider removing this test as it does not verify any logic and needs to be ignored.